### PR TITLE
Added cli command to show config

### PR DESCRIPTION
**Pull request by emulbreh**

### CODE REVIEW

File: lymph/cli/config.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+class ConfigCommand(Command):
+    """
+    Usage: lymph config [options]
+
+    Outputs the configuration.
+
+    {COMMON_OPTIONS}
+    """
+    def run(self):
+        print yaml.safe_dump(self.config.values)
```
Comment (mouadino) [inline lymph/cli/config.py]: pprint ?

Comment (emulbreh) [inline lymph/cli/config.py]: that's yaml output.

File: lymph/tests/integration/test_cli.py [hunk 2]
```diff
@@ -45,6 +45,7 @@ def test_negative_inspect(self):
 class ListCommandTests(CliTestMixin, unittest.TestCase):
     def test_list(self):
         self.assert_lines_equal(['list'], u"""
+            {t.bold}config     {t.normal}
```
Comment (mamachanko) [inline lymph/tests/integration/test_cli.py]: description?

Comment (emulbreh) [inline lymph/tests/integration/test_cli.py]: okok

**Comment by mamachanko** [comment]
+1

**Pull request merged**